features/facebookImport/src/components/dataStructureMiniStory/dataStructureMiniStory.jsx

```diff
@@ -15,12 +15,24 @@ const DataStructureMiniStory = ({ data }) => {
         return b.value - a.value;
     });
 
+    let totalFiles = 0;
+    data.forEach((a) => (totalFiles += a.count));
+
     const bubbleVizWidth = 400;
     const bubbleVizHeight = 400;
     const dataBubblesDarkColor = "#0f1938";
     const dataBubblesLightColor = "#f7fafc";
     const [selectedFolder, setSelectedFolder] = useState(data[0].title);
 
+    const extendedDataStructure = [
```

Comment:
Nit: I had to wonder a bit about what this is, how about a name like `dataWithTotal` or something? Not my best idea, but at least it's kinda clear then what it is :D